components/slider/src/mark/label/use-mark-label.ts

```diff
@@ -14,6 +14,10 @@ export const useMarkLabel_unstable = (
   const minValue = values.length > 1 ? Math.min(...values) : 0;
   const maxValue = Math.max(...values);
 
+  const active = props.activeEqual
+    ? minValue === props.value || maxValue === props.value
```

Comment:
This needs to check all possible values, since sliders with 3 or more values are also supported.

```suggestion
    ? values.some((value) => value === props.value)
```

Reply:
Aha, did not think about more than two. Updated the gradient logic to work with 3 or more.
![image](https://github.com/user-attachments/assets/84315a86-886e-42db-98ef-e514e9bfe0ba)